src/stories/Changelog.stories.mdx

```diff
@@ -3,8 +3,12 @@ import { Meta } from "@storybook/addon-docs";
 <Meta title="Changelog" />
 # Changelog
 
-## 0.18.0
-- Added Collapsible component to Storybook
```

Comment:
These lines shouldn't be deleted. We're losing the specific changelog for 0.18.0 version